sklearn/datasets/openml.py

```diff
@@ -438,6 +439,9 @@ def fetch_openml(name=None, version='active', data_id=None, data_home=None,
         Specify another download and cache folder for the data sets. By default
         all scikit-learn data is stored in '~/scikit_learn_data' subfolders.
 
+    ignore_strings : boolean, default=True
```

Comment:
I would use default=False since I would like people to use DataFrame and force them to turn this to True if they really want to only use numerical data. It avoids silent errors if they discard the warning.